tests/torch/test_compile_torch.py

```diff
@@ -675,6 +675,9 @@ def test_compile_brevitas_qat(
     )
 
 
+# Due to a Concrete issue, this test is ignored
```

Comment:
strictly speaking, it's not a Concrete issue, it's: Concrete had a bug in simulation which has been fixed, and now, we need to update our test to be compatible with the fixed CP